src/js/components/Calendar/__tests__/Calendar-test.js

```diff
@@ -185,6 +195,7 @@ describe('Calendar', () => {
     );
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
+    component.unmount();
```

Comment:
Is there a configuration on the suit level that can unmount the component after each test run instead of explicitly unmounting it at the end of each test?

Reply:
Within the testing suites there is a combination of tests that use `render` and tests that use `renderer.create` so unmounting the component on all the tests would cause an error in tests that use `render`